src/components/Banners/YatBanner.tsx

```diff
@@ -13,12 +17,34 @@ export const YatBanner: React.FC<YatBannerProps> = ({ isCompact, ...rest }) => {
   const [isLargerThan2xl] = useMediaQuery(`(min-width: ${breakpoints['2xl']})`, { ssr: false })
   const translate = useTranslate()
 
+  // we don't have UI/UX for switching accounts when clicking the yat banner
+  // account 0 is gonna have to cut it
+  const firstEvmAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, ethChainId),
+  )
+
+  const firstEvmAddress = useMemo(() => {
+    if (!firstEvmAccountId) return undefined
+    const { account } = fromAccountId(firstEvmAccountId)
+    return account
+  }, [firstEvmAccountId])
+
+  // spec https://github.com/shapeshift/web/issues/4604
+  const href = useMemo(() => {
+    const baseUrl = 'https://www.y03btrk.com/DFBHL/7XDN2/'
+    if (!firstEvmAddress) return baseUrl
```

Comment:
Tested by switching to Keplr, confirmed we don't have dangling EVM AccountId and we return the base URL (we would have bigger problems if we did)
<img width="319" alt="image" src="https://github.com/shapeshift/web/assets/17035424/8a58ad14-0385-491f-9d22-0b78449bd87c">